general/base_import/import_faq.rst

```diff
@@ -0,0 +1,201 @@
+=======================
+Import CSV file to Odoo
+=======================
+
+
+
+Frequently Asked Questions
+--------------------------
+
+
+Need to import data from an other application?
```

Comment:
missing interrogative word. "How to import data from another application?" or something like that